### DISCUSSION TITLE
I am looking for regular expression code to delete apostrophes at the beginning and end of sentences.

### DISCUSSION THREAD

**Discussion by aylardananil** [SubtitleEdit/subtitleedit#8365] [Q&A]
Hi. I am looking for regular expression code to delete apostrophes at the beginning and end of sentences.
Example:
Before:
'You're beautiful, guys. Don't change a thing.'
After:
You're beautiful, guys. Don't change a thing.

**Comment by niksedk** [comment]
Something like this?

`^'` and `'$`

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/f119f9e4-e61e-4814-a4ac-303b6615fc54)

**Reply by sangacanga1** [reply]
Hi, I'm looking for something like this.
Example:
Before:
So, what's with the 'Bang-bang?'
After:
So, what's with the Bang-bang?
